src/helpers/contacts.spec.js

```diff
@@ -157,3 +158,73 @@ describe('getConnectedAccounts', () => {
     expect(getConnectedAccounts(contact).length).toBe(0)
   })
 })
+
+describe('normalizeFields', () => {
+  it('should create an array [{type: string, values: arrayOfObject}] from contact attributes', () => {
+    const contact = {
+      _id: 'c6899688-6cc6-4ffb-82d4-ab9f9b82c582',
+      id: 'c6899688-6cc6-4ffb-82d4-ab9f9b82c582',
+      _rev: '1-9368a4f2e467c449f4a1f5171a784aa8',
+      _type: DOCTYPE_CONTACTS,
+      metadata: {
+        version: 1
+      },
+      groups: ['abc', 'def'],
+      address: [
+        {
+          city: 'Stafford',
+          country: 'Solomon Islands',
+          postcode: '06635',
+          primary: true,
+          street: '48 Fuller Road'
+        }
+      ],
+      birthday: '1998-06-03',
+      email: [
+        { address: 'rikki.white@celmax.name', primary: true },
+        { address: 'eleanore.fennell@thermolock.name', primary: false }
+      ],
+      name: { familyName: 'White', givenName: 'Rikki' },
+      fullname: 'Rikki White',
+      phone: [
+        { number: '+33 (1)9 14 02 28 31', primary: true },
+        { number: '+33 (2)3 99 53 65 21', primary: false }
+      ],
+      unknownField: 'unknownField',
+      emptyField: ''
+    }
```

Comment:
There is no `Contact Fixture` we can reuse? Instead of creating a new one? If we're adding new attributes to the contact, we'll need to upgrade all our "faked contact"